inspirehep/modules/workflows/tasks/arxiv.py

```diff
@@ -40,6 +40,7 @@
 from inspire_dojson.hep import hep
 from inspire_schemas.utils import classify_field
 from inspirehep.modules.workflows.utils import convert
+from inspire_schemas.builders import LiteratureBuilder
```

Comment:
Nit: same thing, between `inspire_dojson` and the other `inspire_schemas` one.

Reply:
changed.